lib/Plack/Middleware/Session.pm

```diff
@@ -238,6 +238,10 @@ Tatsuhiko Miyagawa
 
 Stevan Little E<lt>stevan.little@iinteractive.comE<gt>
 
+=head1 REPOSITORY
```

Comment:
repository info is already in META info and can be visible on search.cpan and metacpan, I'd avoid repeating that kind of info in multiple places.